homeassistant/helpers/recorder.py

```diff
@@ -0,0 +1,15 @@
+"""Helpers to check the recorder."""
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_start_setup
+
+DATA_INSTANCE = "recorder_instance"
+RECORDER_DOMAIN = "recorder"
+RECORDER_BASE_SETUP_TIMEOUT = 60
+
+
+async def async_wait_for_recorder_full_startup(hass: HomeAssistant) -> None:
```

Comment:
This will go away in the next pr https://github.com/home-assistant/core/pull/49036